src/banks.json

```diff
@@ -57,7 +57,7 @@
   "ABNA": {
     "code": "ABNA",
     "type": "Foreign",
-    "ifsc": null,
+    "ifsc": "NA",
```

Comment:
not so json, can we avoid this

Reply:
Why? null = this value is not set.

Better than an invalid string, imo. Already doing it for MICR/IINs

Reply:
Just saw the diff is opposite. I want `null`, just to be clear. This was done manually in last release, so automating this release.